### PR TITLE
feat: Add demandRankDisplayText to Artwork Price Insights

### PR DESCRIPTION
**Pull request by MrSltun**
This PR adds `demandRankDisplayText` to the artwork price insights

### Example
- Query 
```graphql
query {
  artwork(id: "artwork-id") {
    artistNames
    title
    marketPriceInsights {
      demandRank
      demandRankDisplayText
    }
  }
}
```

- Result
```json
{
  "data": {
    "artwork": {
      "artistNames": "FamousArtist",
      "title": "CoolWork",
      "marketPriceInsights": {
        "demandRank": 0.877,
        "demandRankDisplayText": "Active Demand"
      }
    }
  }
}
```

### CODE REVIEW

File: src/lib/getDemandRank.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/**
+ *
+ * @param demandRank
+ * @returns the demand rank in a textual form
+ * @example getDemandRank(0.2) => "Less Active Demand"
+ * @example getDemandRank(0.6) => "Active Demand"
+ */
+export function getDemandRank(demandRank: number): string {
```
Comment (MounirDhahri) [inline src/lib/getDemandRank.ts:8]: maybe getDemandRankDisplayText? also for the file name

File: src/schema/v2/artwork/index.ts [hunk 2]
```diff
@@ -106,6 +107,11 @@ const ArtworkPriceInsightsType = new GraphQLObjectType<any, ResolverContext>({
     demandRank: {
       type: GraphQLFloat,
     },
+    demandRankDisplayText: {
+      type: GraphQLString,
+      description: "The demand rank  of the artist and medium",
```
Comment (MounirDhahri) [inline src/schema/v2/artwork/index.ts:112]:
```suggestion
      description: "The demand rank  display of the artist and medium",
```

**Review by MounirDhahri — CHANGES_REQUESTED** [review]

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Thank you 🙏

**Review by olerichter00 — COMMENTED** [review]
🌟

**Pull request merged**